front/lib/models.ts

```diff
@@ -446,13 +447,23 @@ Key.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    isSystem: {
```

Comment:
DB-layer logic looks good to me, but we need to hide these from the user-visible list of API keys.